version.txt

```diff
@@ -0,0 +1 @@
+v0.0.0-0-0
```

Comment:
can we make the version simpler? just x.y.z

Reply:
see vcversioner documentation